postgres/datadog_checks/postgres/util.py

```diff
@@ -841,3 +841,35 @@ def get_list_chunks(lst, n):
         {'name': 'postgresql.subscription.sync_error', 'type': 'monotonic_count'},
     ],
 }
+
+# Requires PG16+
+STAT_IO_METRICS = {
+    'name': 'stat_io_metrics',
+    'query': """
+SELECT backend_type,
+       object,
+       context,
+       reads,
+       read_time,
+       writes,
+       write_time,
+       extends,
+       extend_time,
+       fsyncs,
+       fsync_time
+FROM pg_stat_io
+""",
+    'columns': [
```

Comment:
@bonnefoa Thank you for the review! For the first pass I was trying to grab the metrics that to me seemed most helpful. There seemed like a bunch of applicable metrics under pg_stat_io and I was trying to avoid initially adding everything if they weren't as useful. I'm happy to add hits and evictions as well if they have high interest.

If you have additional thoughts and insights on which of these metrics are the most useful and should be added I'd love to hear more, either here or can grab a few minutes of your time and chat!

Reply:
Added hits and evictions in [b5ce4d1](https://github.com/DataDog/integrations-core/pull/17423/commits/b5ce4d12a484a1fad1f6dd4fa09a673701c43aa6).